.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,23 @@
+name: Lint
+
+env:
+  SHELLCHECK_OPTS: -e SC1008 -s bash
```

Comment:
Unless you expect this to change, it can be moved down to under the specific step